recipes/libignition-transport/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set base_name = "libignition-transport" %}
+{% set version = "4.0.0" %}
+{% set name = base_name + version.split('.')[0] %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/ignitionrobotics/ign-transport/get/ignition-transport{{ version.split('.')[0] }}_{{ version }}.tar.gz
+    sha256: 43d46b8eaed2ca4ace57591bb2f4d053513f527f6fd876516daa52d124d8dc6c
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
+    - vs2017_win-32                      # [win32]
+    - make                               # [not win]
+    - cmake
+    - pkg-config
+  host:
+    - libignition-msgs1
+    - cppzmq
+    - zeromq
+    - libuuid                            # [not win]
+    - libprotobuf                        # [not win]
+  run:
+    - libignition-msgs1
+    - cppzmq
+    - zeromq
+    - libuuid                            # [not win]
+    - libprotobuf                        # [not win]
+
+test:
+  commands:
+    - if exist %PREFIX%\\Library\\lib\\cmake\\ignition-transport4\\ignition-transport4-config.cmake (exit 0) else (exit 1)  # [win]
```

Comment:
Can you add a test for unix as well?

Reply:
Also add tests for the existence of a header file and the shared library.

Reply:
Sure, just added them.